employee-rostering-frontend/src/setupTests.ts

```diff
@@ -54,3 +54,4 @@ function mockFunctions() {
 }
 
 jest.mock('react-responsive', () => mockFunctions());
+jest.mock('util/BookmarkableUtils');
```

Comment:
It's mocked-out here so it's actually not used by tests.

https://jestjs.io/docs/en/manual-mocks

Reply:
But I unmocked it in `BookmarkableUtils.test.ts:
```
describe('Bookmarkable Utils', () => {
  beforeAll(() => {
    jest.unmock("util/BookmarkableUtils");
  });
  // ...
```